docs/reference-manual/processes/library/ai-recognition.md

```diff
@@ -11,50 +11,55 @@ permalink: /processes/ai-recognition.html
 ---
 # AI Recognition
 
-![AI Recognition]({{ site.img }}/reference/processes/ai-recognition/ai-recognition.png "AI Recognition") 
+How to run ONNX models in score
```

Comment:
I think this sentence was good as a title but a bit weird like this without punctuation